homeassistant/components/nest/api.py

```diff
@@ -16,20 +20,37 @@ def __init__(
         self,
         websession: ClientSession,
         oauth_session: config_entry_oauth2_flow.OAuth2Session,
-        api_url: str,
+        client_id: str,
+        client_secret: str,
     ):
         """Initialize Google Nest Device Access auth."""
-        super().__init__(websession, api_url)
+        super().__init__(websession, API_URL)
         self._oauth_session = oauth_session
+        self._client_id = client_id
+        self._client_secret = client_secret
 
     async def async_get_access_token(self):
```

Comment:
Is the library still using this method?

Reply:
As long as Home Assistant can still update the token as needed when setting up the integration, it seems ok.